src/Presentation/Nop.Web/Areas/Admin/Factories/ProductModelFactory.cs

```diff
@@ -779,10 +782,17 @@ public virtual async Task<ProductListModel> PrepareProductListModelAsync(Product
                 categoryIds.AddRange(childCategoryIds);
             }
 
+            var minimumPrice = searchModel.MinimumPrice == 0 && searchModel.MaximumPrice == 0
+                ? await _productService.GetMinPrice() : searchModel.MinimumPrice;
```

Comment:
instead of getting GetMinPrice(), just set this to null please, so in this way you will say, "dont filter by minPrice"